src/shared/times-option.js

```diff
@@ -0,0 +1,91 @@
+/// <reference path="./types.js" />
+
+/**
+ * @module shared
+ */
+
+/**
+ * @type AriaATCIShared.timesOption
+ */
+export const timesOption = {
+  afterNav: 1000,
+  afterKeys: 5000,
+  testSetup: 1000,
+  modeSwitch: 750,
+  docReady: 2000,
+};
```

Comment:
Stateful modules are a source of tech debt in my experience. This clearly works for our current needs, but at some point in the future (e.g. when we go to write a new kind of test or re-use something), the `timesOption` singleton may make the change more involved than if we had explicitly passed the value around to begin with. Think we can treat these values a little more like the rest of the command-line arguments?

Reply:
It's possible, the reason I was sort of avoiding it is this would necessitate passing these options into modules everywhere, I feel like the usability/dev experience of adding a timeout to it in the future might be pretty large.  I'll do this as a separate commit to kind of weigh it out (not having done it yet, it might not be "that bad")

Reply:
In the end this didn't feel too terrible. ea8d351